apps/judicial-system/web/src/components/AppealAlertBanner/AppealAlertBanner.strings.ts

```diff
@@ -0,0 +1,30 @@
+import { defineMessages } from 'react-intl'
+
+export const strings = defineMessages({
+  appealTitle: {
+    id: 'judicial.system.core:appeal_alert_banner.appeal_title',
+    defaultMessage: 'Kærufrestur rennur út {appealDeadline}',
```

Comment:
Since you can appeal after the appeal deadline has passed, I think we need to be able to say 

Kærufrestur rann út {appealDeadline}

Reply:
Discussed offline - this will be handled in separate task